ui/src/settings.ts

```diff
@@ -122,6 +122,14 @@ const PRESETS: Preset[] = [
     declutterEnabled: true,
     equationDiagramsEnabled: true,
   },
+  {
+    key: "sab-2021",
+    primerInstructionsEnabled: false,
+    termGlossesEnabled: true,
+    declutterEnabled: true,
+    symbolUnderlineMethod: "defined-symbols",
```

Comment:
This is minor, but it could be helpful to add an enum for these `symbolUnderlineMethod` types since they're used a few times throughout this file.

Reply:
Thank you @carolinepaulic ! This is a great idea and in fact there is already a type defined for this property in the settings type declaration:

https://github.com/allenai/scholarphi/blob/b9c50f2b406e65752aad142cf8d44275a11ea662/ui/src/settings.ts#L51

https://github.com/allenai/scholarphi/blob/b9c50f2b406e65752aad142cf8d44275a11ea662/ui/src/components/entity/EntityAnnotationLayer.tsx#L23

This is not an enum per se though it does check the string. Would there be advantage to using an enum type?